src/app/exercise/test-description/test-description.component.ts

```diff
@@ -1,17 +1,22 @@
-import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
 import {FileConfig} from '../interfaces/file-config';
 
 @Component({
   selector: 'slides-test-description',
   templateUrl: './test-description.component.html',
   styleUrls: ['./test-description.component.css']
 })
-export class TestDescriptionComponent {
+export class TestDescriptionComponent implements OnInit {
   @Input() title: string;
-  @Input() file: FileConfig;
+  @Input() filePath: string;
   @Input() pass: boolean;
   @Output() onSelectFile = new EventEmitter<FileConfig>();
 
   constructor() {
   }
+  ngOnInit() {
+    if (!this.title) {
+      console.error('Test description is missing');
```

Comment:
Rename the variable to "description" or change the error to "Test title is missing".